README.md

```diff
@@ -3,6 +3,9 @@
 
 Helm charts for the [Brigade](https://github.com/brigadecore/brigade) project.
 
+**Note**: the charts in this repo are for Brigade v1.  For Brigade v2 charts,
+see the [charts][v2 charts] directory in the [v2 branch].
```

Comment:
imho, we should make it more clear that the v2 branch in question belongs to the main brigade repo and not this repo. Something like:

> For Brigade v2 charts, see the [charts][v2 charts] directory in the [v2 branch] of the main brigadecore/brigade repository.

Or something to that effect.